tests/api/v1/subjects/post.js

```diff
@@ -142,6 +142,8 @@ describe(`tests/api/v1/subjects/post.js, POST ${path} >`, () => {
         const result = JSON.parse(res.text);
         expect(Object.keys(result)).to.contain('parentAbsolutePath');
         expect(result.parentAbsolutePath).to.equal.null;
+        expect(result).to.have.property('user');
```

Comment:
How is owner different than user here? _result_ has both owner and user with exactly same value. This probably applies to other endpoints as well.

Reply:
@pallavi2209 we have a user story "#spike figure out how we should clarify/clean up user associations #rf" to address this concern.